toolbox/util/githubUtils.go

```diff
@@ -124,22 +125,30 @@ func GetAPITokenFromFile(tokenFile string) (string, error) {
 // CloneRepoCheckoutBranch removes previous repo, clone to local machine,
 // change directory into the repo, and checkout the given branch.
 // Returns the absolute path to repo root
-func CloneRepoCheckoutBranch(gclient *GithubClient, repo, baseBranch, newBranch string) (string, error) {
-	if err := os.RemoveAll(repo); err != nil {
+func CloneRepoCheckoutBranch(gclient *GithubClient, repo, baseBranch, newBranch, pathPrefix string) (string, error) {
+	if err := os.MkdirAll(pathPrefix, os.FileMode(0755)); err != nil {
 		return "", err
 	}
-	if _, err := ShellSilent(
-		"git clone " + gclient.Remote(repo)); err != nil {
+	repoPath := path.Join(pathPrefix, repo)
+	if err := os.RemoveAll(repoPath); err != nil {
+		return "", err
+	}
+	if err := os.Chdir(pathPrefix); err != nil {
+		return "", err
+	}
+	var additionalEnv []string //no additional environment variables
```

Comment:
Is there a significance with using using "additionalEnv" here but "nil" elsewhere?  I'd be inclined to just use nil, or have kept Shell() as-in but added a ShellWithEnv() that's used by the one instance I saw that cares to pass in an environment variable.